tests/unit/test_queries.py

```diff
@@ -18,22 +20,28 @@
 class TestQueries(unittest.TestCase):
     def setUp(self):
         # Opensearch client Params
-        os.environ[
-            "OS_DOMAIN"
-        ] = "search-sds-metadata-uum2vnbdbqbnh7qnbde6t74xim.us-west-2.es.amazonaws.com"
+        # TODO: there has to be a better way
```

Comment:
Nice, I agree with this comment :)

With your new domain work here I would think this could also get routed through a base domain name for simpler access?